src/components/FlawAffects.vue

```diff
@@ -424,10 +426,49 @@ function selectAffects(event: Event) {
 }
 
 // Affects Fields
-function affectCvss3Vector(affect: ZodAffectType) {
-  affect.cvss_scores.find(({ issuer, cvss_version }) => issuer === 'RH' && cvss_version === 'V3')
-    ?.vector
-    || null;
+function affectCvss(affect: ZodAffectType) {
+  return affect.cvss_scores.find(({ issuer, cvss_version }) => issuer === 'RH' && cvss_version === 'V3');
+}
+
+function affectCvssDisplay(affect: ZodAffectType) {
+  const cvssScore = affectCvss(affect)?.score;
+  const cvssVector = useAffectCvss3Vector(affect).value;
+  if (cvssScore && cvssVector) {
+    return `${affectCvss(affect)?.score || ''} ${useAffectCvss3Vector(affect).value}`;
+  } else {
+    return useAffectCvss3Vector(affect).value;
+  }
+}
+
+function useAffectCvss3Vector(affect: ZodAffectType) {
+  return computed({
+    get() {
+      return affectCvss(affect)?.vector || '';
+    },
+    set(newValue: string) {
+      const cvssScoreIndex = affect.cvss_scores.findIndex(
+        ({ issuer, cvss_version }) =>
+          issuer === 'RH' && cvss_version === 'V3'
+      );
+      const cvssId = affect.cvss_scores[cvssScoreIndex].uuid;
+      if (newValue === '' && cvssScoreIndex !== -1 && affect.uuid && cvssId) {
+        affect.cvss_scores.splice(cvssScoreIndex, 1);
+        emit('affect:removeCvss', affect.uuid, cvssId);
+      } else if (cvssScoreIndex !== -1) {
+        affect.cvss_scores[cvssScoreIndex].vector = newValue;
+      } else if (newValue !== '') {
+        affect.cvss_scores.push({
+          issuer: 'RH',
+          cvss_version: 'V3',
+          comment: '',
+          score: null,
+          vector: newValue,
+          embargoed: props.embargoed,
+          alerts: [],
+        });
```

Comment:
Mutating props is a bad practice, in Vue props are **one way only**, this should be changed to an emit and handled in the parent
[One way data flow](https://vuejs.org/guide/components/props.html#one-way-data-flow)
[Mutating object/array props](https://vuejs.org/guide/components/props.html#mutating-object-array-props)